### ISSUE TITLE
Refresh doesn't work?

### ISSUE THREAD

**Issue by YanDevDe** [fauna/vscode#74]
Hello everyone!

I've been trying to use this extension under fauna-dev (which runs per docker). Hence I'm using .faunarc at my root project directory with empty global admin key.

I've following content at `.faunarc`

```
FAUNA_KEY=secret
FAUNA_DOMAIN=127.0.0.1
FAUNA_SCHEME=http
FAUNA_PORT=8443
FAUNA_GRAPHQL_HOST=http://localhost:8084/graphql
```

This config works fine on `fauna-schema-migration` and `faunadb` nodejs module - my test runs smoothly and it can pull data from fauna-dev too.

But for some reason, when I added some data to databases - I don't see this at FaunaDB Plugins in VSCode. If I click refresh, it just happens nothing. Not even loader thing appears.

Temporary fix would be closing Visual Studio Code and re-open it again and then the data are visible - which kinda... sucks lol

Am I missing something here?

Been running fauna's docker by:

```bash
docker run --rm --name faunadb -p 8443:8443 -p 8084:8084 fauna/faunadb
```

Any tips or help are welcome! :)

Thanks,

Yan

**Comment by fireridlle** [comment]
Don't you have any error messages at the bottom right corner?
Could you try following and describe what you get

1. create a collection
```
CreateCollection({name: "test"})
```
2. click the refresh button. does `test` collection appear at the right sidebar? if no, try step 3
3. run paginate collection query. does `test` collection present at the response?
```
Paginate(Collections())
```

**Comment by YanDevDe** [comment]
The refresh did nothing, so yeah, I get no error messages either too.

![image](https://user-images.githubusercontent.com/1526048/119347131-5c559f00-bc9b-11eb-817f-85d6f3180322.png)

Looks like I'm getting timeout issue (same for `Paginate(Collections())`). 

However the IP-Address (http://127.0.0.1:8443/) is working normally on `faunadb`, but at VSCode Extension it isn't working?

The configuration is the same which I uses for `faunadb` too:

```bash
# .env file for nodejs module "faunadb"
FAUNA_ADMIN_KEY=secret
FAUNADB_DOMAIN=127.0.0.1
FAUNADB_SCHEME=http
FAUNADB_PORT=8443
```

**Comment by YanDevDe** [comment]
The refresh did nothing, so yeah, I get no error messages either too.

I can add normally with `CreateCollection({name: "test"})`and I can receive `Paginate(Collections())`, but the refresh still doesn't work, it just... do absolutely nothing.

![image](https://user-images.githubusercontent.com/1526048/119348668-798b6d00-bc9d-11eb-815f-19c3bd5c8280.png)

After I entered everything, I clicked on refresh icon and it does nothing again. It only show once when Visual Studio Code has started for the first time after closing it.

**Comment by YanDevDe** [comment]
When I restart VSCode, then this is shown 

![image](https://user-images.githubusercontent.com/1526048/119349081-02a2a400-bc9e-11eb-952a-1b63e3c7b3f1.png)

However, it doesn't restart still + only works when running VSCode as new window.

**Comment by YanDevDe** [comment]
Does it has something to do with GRAPHQL? I don't use GraphQL so I'm not quite sure if this is the issue.

```
FAUNA_GRAPHQL_HOST=http://localhost:8084/graphql
```

**Comment by fireridlle** [comment]
@YanDevDe have you deleted databases login.js1, login.js2 etc..? are not shown on the second screen.
have you faced the same issue with db.fauna.com?

**Comment by YanDevDe** [comment]
Yes, I've deleted login.js1 etc because this was from test (and it auto-removed again in test). Sorry for the confusion.

db.fauna.com with following `.faunarc`

```bash
FAUNA_ADMIN_KEY=<fauna secret key from fauna.com>
```

Also lists the database (with demo data) after VSCode restart - but it doesn't refresh here either too.

![image](https://user-images.githubusercontent.com/1526048/119350663-e4d63e80-bc9f-11eb-8910-a870d624ce80.png)

**Comment by hugomrdias** [comment]
I have probably the same issue, refresh doesnt show newly created documents
